injector.js

```diff
@@ -62,17 +62,17 @@ Injector.DEFAULT_CONTEXT = '_';
  *     'moduleB': 'mock/moduleB'
  *   });
  */
-Injector.prototype.map = function(id, mockId) {
+Injector.prototype.map = function (id, mockId) {
   // Support object notation
-  if (id && typeof id === 'object') {
-    _.each(id, function(value, key) {
+  if (id && _.isObject(id)) {
```

Comment:
`null` protection is not needed with lodash method

```suggestion
  if (_.isObject(id)) {
```